spring-cloud-gcp-samples/spring-cloud-gcp-data-spanner-repository-sample/README.adoc

```diff
@@ -1,13 +1,12 @@
-= Spring Cloud GCP Cloud Spanner Starter Example
+= Spring Cloud GCP Cloud Spanner Starter Example using "Repository"
 
-This code sample demonstrates how to read and write POJOs from Google Cloud Spanner using the Spring
-Data Cloud Spanner module link:../../spring-cloud-gcp-starters/spring-cloud-gcp-starter-data-spanner[Spring Cloud GCP Cloud Spanner Starter].
+This code sample demonstrates how to read and write POJOs from Google Cloud Spanner using the Spring Data Cloud Spanner module link:../../spring-cloud-gcp-starters/spring-cloud-gcp-starter-data-spanner[Spring Cloud GCP Cloud Spanner Starter].
```

Comment:
You can say something like "using Spring Data repositories" and link to Spring Data documentation for concepts overview: https://docs.spring.io/spring-data/data-commons/docs/2.5.x/reference/html/#repositories.core-concepts